powerauth-restful-security-base/src/main/java/io/getlime/security/powerauth/rest/api/base/provider/PowerAuthEncryptionProviderBase.java

```diff
@@ -34,6 +44,9 @@
  */
 public abstract class PowerAuthEncryptionProviderBase {
 
+    private ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
... this is actually the first giveaway that something is wrong - this class should not depend on `ObjectMapper`...

Reply:
What do you suggest? There are three usages of `ObjectMapper` in this class.